lc0_main.go

```diff
@@ -299,6 +300,31 @@ func checkLc0() {
 	}
 }
 
+func checkDx(networkPath string) {
+	dir, _ := os.Getwd()
+	if !hasBlas {
+		log.Fatalf("Dx12 backend cannot be validated")
+	}
+	log.Println("Sanity checking the dx12 driver.")
+	cmd := exec.Command(path.Join(dir, "lc0"))
+	sGpu := ""
+	if *gpu >= 0 {
+		sGpu = fmt.Sprintf(",gpu=%v", *gpu)
+	}
+	cmd.Args = append(cmd.Args, "benchmark", "-w", networkPath, "--backend=check")
+	cmd.Args = append(cmd.Args, fmt.Sprintf("--backend-opts=mode=check,freq=1.0,atol=5e-1,dx12%v", sGpu))
+	// Add the startpos fen to get consistent behavior with old and new lc0 benchmark.
+	cmd.Args = append(cmd.Args, "--fen=rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1")
```

Comment:
I would be almost tempted to just use the new behavior - multiple positions for testing seems like an improvement...

Reply:
It takes almost 6 minutes to run, so probably not the full test, but we can run a couple more positions, maybe for 5 seconds each instead of ten.

Reply:
ahh 6 minutes would be bad - 15 or 20 seconds might be acceptable.